### ISSUE TITLE
Version number not updated

### ISSUE THREAD

**Issue by Pohli** [mentaljam/harbour-storeman#88]
If I start Storeman and go directly from Recently updated to a recently updated app's details page before the automatic update process of repos is finished the available version number displayed is wrong (even after waiting for finish of automatic repos update process and after selecting Refresh cache and Reload from pulley menu). And even after successfully updating that app which leads to a higher installed version number than available version number:

![screenshot_20190216_001](https://user-images.githubusercontent.com/26346589/52910832-8094cb00-329d-11e9-9474-19a83797c9ed.png)

Jolla Phone with SFOS 3.0.1.11 and Storeman 0.1.0

**Comment by Olf0** [comment]
While I perceive it as logical, that an outdated "Available version" may be shown until the current repo data has been successfully fetched and processd, I also experienced that outdated "Available version" info persisted (across "Refresh" actions, and also when manually triggering an update of the repo data again per "Refresh cache") with older Storeman versions (pretty sure v0.1.6 exhibited this issue) until Storeman was closed and restarted.  I don't remember exactly how often this happened: sometimes, often or always?  

This seems to be resolved in Storeman 0.1.8 (or 0.1.7?): For me the "Available version" info is now always updated, when expected.
I also have the impression that the "Available version" info is now immediately updated, when the repository of that RPM was successfully updated (in contrast to, "after all repo data has been fetched and processed", which seemed to be the former behaviour).
